playwright.config.ts

```diff
@@ -28,7 +29,53 @@ export default defineConfig({
   /* Opt out of parallel tests on CI. */
   workers: process.env.CI ? 4 : 6,
   /* Reporter to use. See https://playwright.dev/docs/test-reporters */
-  reporter: 'html',
+  reporter: [
+    [
+      './node_modules/playwright-slack-report/dist/src/SlackReporter.js',
```

Comment:
Is it OK to hardcode a path into `node_modules`? Can't it be `import`ed? 🤔

Reply:
it is how library works from official documents 

suppose i can change that in future